### DISCUSSION TITLE
Ability to add Links that open in a new window

### DISCUSSION THREAD

**Discussion by srelbo** [avitorio/outstatic#202] [Ideas]
Hey @avitorio, would it be possible to add support in the authoring tool to add links that open in a new window? We are facing this problem where we want to link to other websites, but not open them in the same window.

Thanks !

**Comment by avitorio** [comment]
Hey @srelbo,

Not in the editor but you can already do that by using this component on the front end: https://github.com/avitorio/outstatic/blob/canary/examples/advanced-blog/src/components/mdx/custom-link.tsx

If you check the code on our advanced-blog repository, you can replace links on the frontend and they will automatically use next/link if internal links starting with `/` and external links starting with `https://` would  open in a new tab.

**Reply by avitorio** [reply]
You'd need to be using an mdx parser to transform anchor tags into components btw. Let me know if you need any help.
